releasenotes/notes/annotated-params-116288d5628f7ee8.yaml

```diff
@@ -20,4 +21,10 @@ fixes:
     The :class:`.SGate` and :class:`.SdgGate` now correctly return a 
     :class:`.CSGate`, resp. :class:`.CSdgGate`, if they are controlled on
     a single control qubit.
-
+upgrade_circuits:
+  - |
+    The ``annotated`` argument of the :meth:`.Gate.control` method is now 
+    ``None``, which allows Qiskit to choose whether to annotate a controlled operation.
+    Generally, the concrete implementation (``annotated=False``) is returned, if it
+    is available, otherwise we set ``annotated=True``. This allows, for example, to 
+    defer the synthesis of controlled, parameterized gates.
```

Comment:
```suggestion
    The ``annotated`` argument of the :meth:`.Gate.control` method is now 
    ``None``, which allows Qiskit to choose whether to annotate a controlled operation.
    I the concrete implementation (``annotated=False``) is available, it will be returned by
    default. Otherwise, the annotated implementation will be returned (``annotated=True``). 
    This allows, for example, to defer the synthesis of controlled, parameterized gates.
```